net-proxy/3proxy/3proxy-0.8.9.ebuild

```diff
@@ -8,7 +8,7 @@ HOMEPAGE="http://www.3proxy.ru/"
 SRC_URI="https://github.com/z3APA3A/3proxy/archive/${PV}.tar.gz -> ${P}.tar.gz"
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~x86 ~amd64 ~ppc"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
```

Comment:
Sorry but you can't just add new arches to KEYWORDS like such. It has to go through a keyword request.